src/shared/logger/activation.ts

```diff
@@ -198,3 +202,63 @@ async function registerLoggerCommands(context: vscode.ExtensionContext): Promise
         )
     )
 }
+
+/**
+ * Deletes the older logs when there are too many or are using too much space.
+ */
+async function cleanLogFiles(): Promise<void> {
+    const logsDir = path.join(ext.context.globalStoragePath, 'logs')
```

Comment:
why not use `getLogPath()` (defined in this file)? or pass `logPath` to this function? That avoids this bit of constructed knowledge, which would become outdated and lead to a potential bug if we ever change the location again.

Reply:
Moved all to logPath as it will be initialized before this function is called.

Reply:
another reason for making it a parameter to `cleanLogFiles` is that this allows tests to work with a test-only directory.